src/rules/converters/space-within-parens.ts

```diff
@@ -0,0 +1,18 @@
+import { RuleConverter } from "../converter";
+
+export const convertSpaceWithinParens: RuleConverter = tslintRule => {
+    let arg = "";
+    if (tslintRule.ruleArguments.length === 1) {
+        arg = "always";
+    } else {
+        arg = "never";
+    }
```

Comment:
Or more simply:

```ts
const arg = tslintRule.ruleArguments.length === 1 ? "always" : "never";
```

...as the temporary `""` is never used and a bit misleading.